CHANGELOG.md

```diff
@@ -1,7 +1,25 @@
 # Version history
 
 <!-- SPDX-License-Identifier: CC0-1.0 -->
-<!-- written in 2019 - 2022 by The Foundation for Public Code <info@publiccode.net> -->
+<!-- SPDX-FileCopyrightText: 2019-2022 The Foundation for Public Code <info@publiccode.net>, https://standard.publiccode.net/AUTHORS -->
+
+## Version 0.4.1
+
+December 5th 2022: 🖳 the twelth draft clarifies document maturity.
```

Comment:
This emoji doesn't render for me :(

Reply:
Not even here on GitHub?

Reply:
looks fine on github: https://github.com/publiccodenet/standard/blob/2257ef60f251df9f52ce0a79b37eeb545b21740e/CHANGELOG.md

Reply:
<img width="582" alt="Screen Shot 2022-12-05 at 15 20 46" src="https://user-images.githubusercontent.com/5113815/205659802-b56833de-8e9c-4cfb-99c8-c8be31ade85a.png">

Reply:
Okay, so I wonder what's up with that? [U+1F5B3](https://unicode-table.com/en/1F5B3/) is in Unicode version 7.0 (2014)

![image](https://user-images.githubusercontent.com/1174254/205663960-803c711f-70ea-4387-8ba4-e2e3bf2cb509.png)